docs/pages/components/popover.md

```diff
@@ -85,7 +85,7 @@ There are four placement options:
     </div>
     <div class="fd-popover__body fd-popover__body--right" aria-hidden="true" id="popoverA2">
         <div style="margin: 20px;">
-            <span class="fd-identifier fd-identifier--xxl fd-identifier--circle fd-identifier--thumbnail" style="background-image: url('https://placeimg.com/400/400/nature');" role="presentation" aria-label="Nature"></span>
+            <span class="fd-avatar fd-avatar--xxl fd-avatar--circle fd-avatar--thumbnail" style="background-image: url('https://placeimg.com/400/400/nature');" role="presentation" aria-label="Nature"></span>
```

Comment:
it doesn't render properly
![Screen Shot 2020-04-23 at 8 18 00 PM](https://user-images.githubusercontent.com/4380815/80161828-9f78a680-859f-11ea-8340-cf6d3d5d98bc.png)